OpenProjects.html

```diff
@@ -13,6 +13,9 @@
               <li><a href="#remove_ub_tests">Remove undefined behavior from tests</a></li>
            </ul>
           <li><a href="http://clang.llvm.org/"><b>Clang</b></a>
+            <ul>
+              <li><a href="#spirv_tablegen">Automatically generate TableGen file for SPIR-V instruction set</a></li>
```

Comment:
Why it is here? Could you change the description to outline the connections with clang? Or maybe it belongs to some other category?

Reply:
I clarified in the description that this project is for the Clang SPIR-V backend.

Reply:
Sorry for dumb question, how is it different from LLVM SPIR-V backend? Can you point me to existing code that is manually generated? I just want to position the project properly.

Reply:
Sorry, that's fair, I'm incorrectly conflating terms. I've been thinking about the project as under the "HLSL in Clang" umbrella but this project doesn't have any frontend work so under LLVM makes sense. The file to replace is: `llvm/lib/Target/SPIRV/SPIRVInstrInfo.td`